tests/Serializer/PayloadSerializerTest.php

```diff
@@ -418,12 +426,16 @@ public function serializeDataProvider(): iterable
         $event = Event::createTransaction(new EventId('fc9442f5aef34234bb22b9a615e30ccd'));
         $event->setSpans([$span1, $span2]);
 
+        $dsc = DynamicSamplingContext::fromHeader('sentry-public_key=public,sentry-trace_id=d49d9bf66f13450b81f65bc51cf49c03,sentry-sample_rate=1');
+        $event->setSdkMetadata('dynamic_sampling_context', $dsc);
```

Comment:
Shouldn't this be a new test case so that we keep covering a transaction event that does not have the SDK metadata set?